### PR TITLE
docs: add `v0.3.2` release notes to the CHANGELOG

### PR DESCRIPTION
**Pull request by jon-whit**
## Description
Add the `v0.3.2` release notes to the CHANGELOG.

## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [ ] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.2] - 2023-01-18
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.1...v0.3.2)
+
+
+### Added
+* OpenTelemetry metrics and tracing `otlp` exporter (#360) - thanks @AlexandreBrg!
```
Comment (adriantam) [inline CHANGELOG.md:17]: do we want to note this and the optimized ListObjects are released under experimental flag?

Comment (jon-whit) [reply CHANGELOG.md:17]: I thought we already did in a prior release?

It's documented here already
https://openfga.dev/docs/getting-started/setup-openfga#experimental-features

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+### Added
+* OpenTelemetry metrics integration with an `otlp` exporter (#360) - thanks @AlexandreBrg!
+
+  To export OpenTelemetry metrics from an OpenFGA instance you can now provide the `otel-metrics` experimental flag along with the `--otel-telemetry-endpoint` and `--otel-telemetry-protocol` flags. For example,
+
+  ```
+  ./openfga run --experimentals=otel-metrics --otel-telemetry-endpoint=127.0.0.1:4317 --otel-telemetry-protocol=http
+  ```
+
+  For more information see the official documentation on [Experimental Features](https://openfga.dev/docs/getting-started/setup-openfga#experimental-features) and [Telemetry](https://openfga.dev/docs/getting-started/setup-openfga#telemetry).
```
Comment (jon-whit) [inline CHANGELOG.md:25]: - [ ] (todo) @jon-whit add the Telemetry section to the OpenFGA Setup OpenFGA page.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
+
+* Type-bound public access support in the optimized ListObjects implementation (when the `list-objects-optimized` experimental feature is enabled) (#444)
+
+### Fixed
+* Tuple validations for models with schema version 1.1 (#446, #457)
```
Comment (rhamzeh) [inline CHANGELOG.md:30]: Can we mention: Evaluate rewrites on nested usersets in the Optimized ListObjects?

File: CHANGELOG.md (hunk 2 above)
Comment (jon-whit) [reply CHANGELOG.md]: https://github.com/openfga/openfga.dev/pull/340 adds a Telemetry section to the Setup OpenFGA page.

File: CHANGELOG.md (hunk 3 above)
Comment (jon-whit) [reply CHANGELOG.md:30]: Yep 👍

Comment (jon-whit) [reply CHANGELOG.md:30]: Done

**Review by rhamzeh — APPROVED** [review]

**Review by craigpastro — APPROVED** [review]
LGTM

**Review by adriantam — APPROVED** [review]

**Pull request merged**